EvidenceApi.Tests/V1/E2ETests/ResidentsTests.cs

```diff
@@ -79,7 +79,7 @@ public async Task SearchResidentsByGroupIdReturns200()
             DatabaseContext.SaveChanges();
 
             // Act
-            var uri = new Uri($"api/v1/residents/search/?&searchQuery=name=&groupId={residentGroupId.GroupId}", UriKind.Relative);
+            var uri = new Uri($"api/v1/residents/search/?&groupId={residentGroupId.GroupId}", UriKind.Relative);
```

Comment:
Knock off the first ampersand here and on the line below.

Reply:
Addressed in [f801e1c](https://github.com/LBHackney-IT/evidence-api/pull/215/commits/f801e1c87c99a696be9c57f1c48f92e7eacb3f78)